templates/etc/default/docker.j2

```diff
@@ -54,5 +54,10 @@ DOCKER_OPTS='{{ docker__tpl_options | join(" ") }}'
 #DOCKER_OPTS="--dns 8.8.8.8 --dns 8.8.4.4"
 {% endif %}
 
+{% if inventory__environment.http_proxy %}
+export http_proxy={{ inventory__environment.http_proxy }}
+export https_proxy={{ inventory__environment.https_proxy }}
+export no_proxy={{ inventory__environment.no_proxy }}
```

Comment:
These variables should be set in the `debops.docker` default variables so that they can be overridden if necessary, like this:
```
docker__env_http_proxy: '{{ ansible_env.http_proxy | d() }}'
docker__env_https_proxy: '{{ ansible_env.https_proxy | d() }}'
docker__env_no_proxy: '{{ ansible_env.no_proxy | d() }}'
```
And then use the default variables in various configuration files. That way the proxy configuration should be relevant to a paritucular host.

The `inventory__environment` variable can be overriden by `inventory__group_environment` and `inventory__host_environment` variables, so using these directly in the roles isn't optimal. The `debops.environment` role takes care to set these variables in `/etc/environment`, from which other roles should use them via `ansible_env` variable.